airbyte-webapp/src/locales/en.json

```diff
@@ -154,6 +154,8 @@
   "connectionForm.nonBreakingChangesPreference.message": "Set how Airbyte handles syncs when it detects a non-breaking schema change in the source",
   "connectionForm.nonBreakingChangesPreference.ignore": "Ignore",
   "connectionForm.nonBreakingChangesPreference.disable": "Pause sync",
+  "connectionForm.breakingChanges.backdrop.message": "We detected some breaking schema updates, please review the changes in your connection header",
+  "connectionForm.nonBreakingChanges.backdrop.message": "We detected some schema updates, please review the changes in your connection header",
```

Comment:
We could put these into one and pass in a variable to decide whether or not to add in the word "breaking"... I didn't feel too strongly on it either way.

Reply:
Yep.  Fine with whatever Andy proposes there.  I just copy/pasted from the designs.